project/main.py

```diff
@@ -0,0 +1,111 @@
+        logging.debug(f"You are in current directory. Path : {path}")
+    else:
+        path = utils.existing_dir(parsed_args.path_to_save_files)
+        logging.debug(f"Path : {path}")
+
+    if parsed_args.clear_path and parsed_args.path_to_save_files and parsed_args.file_name:
+        logging.info("You decide to clear path before work")
+        utils.clear_files_in_path(parsed_args.path_to_save_files, parsed_args.file_name)
+
+    if parsed_args.clear_path:
+        logging.info("Cleared data before generating files")
+
+    if int(parsed_args.multiprocessing) <= 0:
+        logging.error(f"You chose {parsed_args.multiprocessing} number of processes used to create files. Can't be less than 0")
+
+    if parsed_args.path_to_save_files is None and parsed_args.files_count is not None:
+        logging.info("You didn't choose the path dir, so path is current file ")
+
+    if int(parsed_args.files_count) > 0 and (parsed_args.file_name, parsed_args.file_prefix, parsed_args.path_to_save_files, parsed_args.data_lines, parsed_args.data_schema) is not None:
+        logging.debug(f'The number of processes used to create files is {parsed_args.multiprocessing}')
+        files = utils.construct_files(parsed_args.file_name, parsed_args.file_prefix, parsed_args.files_count)
+        workers_number = int(parsed_args.multiprocessing)
+        if workers_number > number_of_processes:
+            logging.info(f"Value of multiprocessing is higher than CPUs in the system. max CPU {number_of_processes} used instead")
+            workers_number = number_of_processes
+        with ThreadPoolExecutor(max_workers=workers_number) as executor:
+            futures = [executor.submit(utils.create_output_file, parsed_args.data_lines, parsed_args.data_schema, parsed_args.path_to_save_files, file) for file in files]
+        logging.info(f"All files created. Created {len(futures)} files")
+
+    if int(parsed_args.files_count) == 0 and (parsed_args.data_lines, parsed_args.data_schema) is not None:
+        logging.info("You choose to generate 0 files, so result is printed without output file, nor file_name, file_prefix or path_to_save_files won't needed")
+        lines = int(parsed_args.data_lines)
+        utils.create_data_without_output_file(lines, parsed_args.data_schema)
+    if int(parsed_args.files_count) < 0:
+        logging.error("Incorrect value!!! There is no possibility to generate {}. Put positive number or zero to "
+                      "generate answer without output files".format(parsed_args.files_count))
+
+    if parsed_args.file_name is not None and parsed_args.file_prefix is not None and int(parsed_args.files_count) > 0 and int(parsed_args.data_lines) > 0 and parsed_args.data_schema is not None:
+        logging.info(f"You generated {parsed_args.files_count} file with {parsed_args.data_lines} lines named: {parsed_args.file_name} with prefix {parsed_args.file_prefix} method")
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
I checked files which your code outputs. Your json file has 1 list and some number of strings in this list:
["{'date': 1284570209.919098, 'name': 'wy0qmtoh-dw0k-dw0k-dw0k-f64doabrx3wq', 'type': 'client', 'animal_type': 'dog', 'age': 53, 'kids_number': 3}", "{'date': 1518484044.062006, 'name': '0ji9vfc4-sgvh-sgvh-sgvh-tzr3cmwq8j2s', 'type': 'client', 'animal_type': 'cat', 'age': 90, 'kids_number': 6}",  ... ]

But you need to have json lines in the file. Please, check "expected results" in the description of the capstone project.